kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/naturesaura/tree_ritual.js

```diff
@@ -465,7 +465,7 @@ onEvent('recipes', (event) => {
             sapling: 'architects_palette:twisted_sapling',
             output: 'naturesaura:shockwave_creator',
             time: 4 * time_multiplier,
-            id: 'naturesaura:shockwave_creator'
+            id: 'naturesaura:shockwave_creator' // todo: this id gets an altered_recipe_indicator, why reuse it here?
```

Comment:
Perhaps @MuteTiefling knows?

Reply:
The default recipe is a 3x3, so the `altered_recipe_indicator` bit is valid. Looks like I just forgot to give it a proper ID of `${id_prefix}shockwave_creator`

Reply:
Would you mind fixing this too @Quezler? 😄

Reply:
**all** un$prefixed nature's aura recipes in this file?

Reply:
Fixed this in #4867. Just merge that into your stuff and you'll be set.